### PR TITLE
Update TLP Indicator Field's type to singleSelect

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Description
This indicator field shouldn't be a free text field - should be single select from predetermined values of `Green`, `Yellow`, `Red`.

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No - Server version 5.5.0 hasn't been released yet.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: IndicatorFields/incidentfield-trafficlightprotocol.json [hunk 1]
```diff
@@ -13,15 +13,19 @@
...
+    "type": "singleSelect",
     "closeForm": false,
     "editForm": true,
     "required": false,
     "script": "",
     "fieldCalcScript": "",
     "neverSetAsRequired": false,
     "isReadOnly": false,
-    "selectValues": null,
+	"selectValues": [
```
Comment (Itay4) [inline IndicatorFields/incidentfield-trafficlightprotocol.json]: fix indentation

**Comment by avidan-H** [comment]
Adding the `docs-approved` label since there isn't any new language for technical writing review.

**Comment by Itay4** [comment]
still need empty changelog though

**Comment by avidan-H** [comment]
@Itay4 I don't think we do - doesn't fail the build.

**Comment by avidan-H** [comment]
Closing this PR and reopening off of purged master branch.

**Pull request closed without merging**